ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -132,7 +132,7 @@ public struct AvatarGroup: View, ConfigurableTokenizedControl {
         let isStackStyle = tokens.style == .stack
 
         let interspace: CGFloat = tokens.interspace
-        let imageSize: CGFloat = tokens.size.size
+        let imageSize: CGFloat = avatarViews[0].contentSize
```

Comment:
Is it ok to access the size like this? What if the avatarViews was empty?

Reply:
That's fair. I've changed it to make the overflow avatar earlier so we can get the content size from that. With the tokens just being stored in the actual control now, there's not really a good way to get the size without going through an actual avatar.